skyuxconfig.json

```diff
@@ -37,6 +37,18 @@
       },
       "searchConfig": {
       },
+      "footer": {
+        "nav": [
+          {
+            "title": "Privacy Policy",
```

Comment:
Should we use `name` instead of `title` to keep it consistent with the existing nav service?
https://github.com/blackbaud/stache2/blob/master/src/app/public/src/modules/nav/nav-link.ts#L2

Reply:
I think we talked and wanted this to match the nav pattern from the omnibar?  I am fine either way, what's easiest for our users?